lib/node_modules/@stdlib/blas/ext/base/scusumkbn/manifest.json

```
@@ -1,40 +1,50 @@
 {
-	"options": {},
-	"fields": [
-		{
-			"field": "src",
-			"resolve": true,
-			"relative": true
-		},
-		{
-			"field": "include",
-			"resolve": true,
-			"relative": true
-		},
-		{
-			"field": "libraries",
-			"resolve": false,
-			"relative": false
-		},
-		{
-			"field": "libpath",
-			"resolve": true,
-			"relative": false
-		}
-	],
-	"confs": [
-		{
-			"src": [
-				"./src/scusumkbn.c"
-			],
-			"include": [
-				"./include"
-			],
-			"libraries": [
-				"-lm"
-			],
-			"libpath": [],
-			"dependencies": []
-		}
-	]
+  "options": {
+    "task":"build"
+  },
+  "fields": [
+   {
+     "field": "src",
+     "resolve": true,
+     "relative": true	
+   },
+   {
+     "field": "include",
+     "resolve": true,
+     "relative": true	
+   },
+   {
+     "field": "libraries",
+     "resolve": false,
+     "relative": false	
+   },
+   {
+     "field": "libpath",
+     "resolve": true,
+     "relative": false	
```

```suggestion
     "relative": false
```

Not clear to me why, but you have trailing tabs.